src/main/java/racing/interfaces/RacingGameController.java

```diff
@@ -17,9 +17,10 @@ public static void main(String[] args) {
 
         ResultView resultView = new ResultView();
         RacingCars racingCars = new RacingCars();
+        RandomMovingStrategy movingStrategy = new RandomMovingStrategy();
```

Comment:
List, Map 등 자료 구조를 선언할 때도 타입은 인터페이스 타입으로 선언하듯이 
이동 전략의 타입을 인터페이스로 선언하면 어떨까요?
```suggestion
        IMovingStrategy movingStrategy = new RandomMovingStrategy();
```